src/tests/test_magic_plot.py

```diff
@@ -491,6 +530,16 @@ def test_hist_breaks(load_penguin, ip):
     )
 
 
+@_cleanup_cm()
+@image_comparison(
+    baseline_images=["hist_binwidth"], extensions=["png"], remove_text=True
+)
+def test_hist_binwidth(load_penguin, ip):
+    ip.run_cell(
+        "%sqlplot histogram --table penguins.csv --column body_mass_g --binwidth 150"
```

Comment:
test for both `--binwidth` and `-W`

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/789/files#diff-991baa7f44b5469ef8e5357e8766a669d91077a9bd5ee583a2a6db58342e14e3R564-R570)